AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -149,6 +149,11 @@ - (void)setPlaceholderImagefromAsset:(AVAsset*)asset
   ASPerformBlockOnBackgroundThread(^{
     ASDN::MutexLocker l(_videoLock);
     
+    // Skip the asset image generation if we don't have any tracks available that are capable of supporting it
```

Comment:
Assuming the comment below starting with // Unfortunately is one of the main cases where this early return is hit, could you move those comment lines up here?  I think it is fine to still have the check for if (image), though.

Also below: it looks like it might be incorrect to check _placeholderImageNode.image == nil.  What if it already has an image but now needs a new one applied to it?

Does kCMTimeZero work for the times array or is 0, 1 necessary?